Makefile

```diff
@@ -3,7 +3,7 @@ COMMIT?=$(shell git describe HEAD)$(shell git diff --quiet || echo '+dirty')
 # Use linker flags to provide commit info
 LDFLAGS=-ldflags "-X=github.com/foundriesio/fioctl/subcommands/version.Commit=$(COMMIT)"
 
-linter:=$(shell which golangci-lint 2>/dev/null || echo $(HOME)/go/bin/golangci-lint)
+linter:=$(shell pwd)/bin/golangci-lint
```

Comment:
Each project requires a specific version of the tool, and as a developer, I prefer not to install or check it in GOPATH/bin, which would make it universally applicable across all projects.

P.S.: $(HOME)/go/bin is not a universal solution for all environments; the correct path would be $GOPATH/bin.

@vkhoroz ^

Reply:
That's just how it works for many years, even before I joined Foundries.io.
It may be not ideal, it just works.
Just IMHO it is logical to add GOPATH into PATH in `.bashrc`, in which case it fits all sizes.